Libplanet/Net/Messages/Message.cs

```diff
@@ -77,6 +78,11 @@ internal enum MessageType : byte
             /// Contains the calculated recent states and state references.
             /// </summary>
             RecentStates = 0x0c,
+
+            /// <summary>
+            /// Message containing a single <see cref="BlockHeader"/>.
+            /// </summary>
+            BlockHeaderMessage = 0x0d,
```

Comment:
I guess we aren't using `Message` postfix for message types. so how about just `BlockHeader`?

Reply:
I've considered it but the class name with `Libplanet.Blocks.BlockHeader` conflicts with `Libplanet.Net.Messages.BlockHeader` and it makes code look nested.